js-peer/src/context/ctx.tsx

```diff
@@ -28,6 +34,7 @@ let loaded = false
 export function AppWrapper({ children }: WrapperProps) {
   const [libp2p, setLibp2p] = useState<Libp2pType | undefined>(undefined)
   const [error, setError] = useState('')
+  const [connections, setConnections] = useState<Connection[]>([])
```

Comment:
If you're going to move connections to the context state here, also move the useEffect hook from the [pages/index.tsx](https://github.com/libp2p/universal-connectivity/pull/210/files#diff-da511c52b901e9c1f5a7c359c4c36eefa3376fe6ac7538242390c660bc47a684R63-R75) and replace other users of connections with the context state.